interactive-bots/helloworld/abhayKashyap03/notes.md

```diff
@@ -0,0 +1 @@
+I had a problem subscribing the 'helloworld' bot to the stream, but it was solved later when I realised that I had created the bot in the actual Zulip website instead of my dev server.
```

Comment:
Please line wrap at 79 characters. This is done so that `git diff` is readable in terminals.
(Simply put: Please break the line for better readability :) )